spec/services/access_request_approval_service_spec.rb

```diff
@@ -81,7 +81,7 @@
         let!(:target_user) do
           create(
             :user,
-            :opted_in,
+            email: access_request.email_address,
```

Comment:
Nice catch


Can change this test as well, as it should of caught it out, 


expect(target_user.organisations).to(
             match_array(target_user.organisations.uniq)
          )

Reply:
Ugh, I messed this up when refactoring the test. This is what happens when rushing. :(

Reply:
This test does that @defong